Rakefile

```diff
@@ -0,0 +1,29 @@
+require "yast/rake"
+
+Yast::Tasks.configuration do |conf|
+  # lets ignore license check for now
+  conf.skip_license_check << /.*/
```

Comment:
Um, I'm not sure actually, maybe not, I just want to be on the safe side. The plan is to [enable the license check](https://trello.com/c/YLWtoKFR/915-todo-run-rake-check-license-at-travis) for all Travis builds.